spec/oncocore.txt

```diff
@@ -31,6 +31,7 @@ Based on:	    CancerCondition
 Description:    "Records the history of the primary cancer condition, in which abnormal cells divide without control and can invade nearby tissues."
                 Code from PrimaryCancerDisorderVS if covered
   	            StageInformation.StageDetail value is type ref(CancerStageGroup)
+0..1			BodyLocation
```

Comment:
Isn't `CancerCondition` supposed to be a "superclass" of `PrimaryCancerCondition` and `SecondaryCancerCondition`? Either can be associated with multiple body locations. (edited)